bin/mycroft-listen

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SOURCE="${BASH_SOURCE[0]}"
+cd -P "$( dirname "$SOURCE" )"
+DIR="$( pwd )"
+
+# Enter the Mycroft venv
+source "$DIR/../venv-activate.sh" -q
+
+# Send a message to be spoken
+output=$(python -m mycroft.messagebus.send "mycroft.mic.listen")
```

Comment:
Since the output isn't used maybe just do
`python -m mycroft.messagebus.send "mycroft.mic.listen" > /dev/null`

Reply:
That makes sense, no reason to store unnecessary data.